### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by reptilbud** [mixer/etcd3#44]
Trying to run ```npm run build:proto``` and got error

```
> etcd3@0.2.5 build:proto /Users/sebastien/Documents/github/mixer
> node ./bin/update-proto ./proto && node bin/generate-methods.js ./proto/rpc.proto > src/rpc.ts

RangeError: Maximum call stack size exceeded
    at String.replace (<anonymous>)
    at stripPackageNameFrom (/Users/sebastien/Documents/github/mixer/bin/generate-methods.js:98:17)
    at MessageCollection.find (/Users/sebastien/Documents/github/mixer/bin/generate-methods.js:63:27)
    at _.values.map.f (/Users/sebastien/Documents/github/mixer/bin/generate-methods.js:233:24)
    at arrayMap (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:660:23)
    at Function.map (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:9571:14)
    at interceptor (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:16970:35)
    at Function.thru (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:8812:14)
    at /Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:4379:28
    at arrayReduce (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:704:21)
    at baseWrapperValue (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:4378:14)
    at LodashWrapper.wrapperValue (/Users/sebastien/Documents/github/mixer/node_modules/lodash/lodash.js:9067:14)

```

I'm trying to do that because I had to change the file ``` bin/template/duplex-stream-method.tmpl ``` to allow add **metadata** when client function are called.

*Notes:* Even without my changes I got the error

**Comment by connor4312** [comment]
Looks like recent changes to etcd3's proto files introduced a dependency cycle in their messages, which we don't handle. Addressing it...

**Comment by connor4312** [comment]
Fixed in 0.2.6.
